server/aws/lb.tf

```diff
@@ -232,3 +232,65 @@ resource "aws_lb_listener" "covidshield_key_submission_test" {
   }
 
 }
+
+###
+# AWS LB - Key Submission Static
+###
```

Comment:
With the amount of duplication we are having for this looks like LB is a good candidate for a TF Module.